docker/README.md

```diff
@@ -0,0 +1,72 @@
+Docker Images
+=============
+
+This directory contains docker image for Kafka.
+The scripts take a url containing kafka as input and generate the respective docker image.
+There are interactive python scripts to release the docker image and promote a release candidate.
+
+Local Setup
+-----------
+Make sure you have python (>= 3.7.x) and java (>= 17) installed before running the tests and scripts.
+
+Run `pip install -r requirements.txt` to get all the requirements for running the scripts.
+
+Bulding image and running tests locally
+---------------------------------------
+- `docker_build_test.py` script builds and tests the docker image.
+- kafka binary tarball url along with image name, tag and type is needed to build the image. For detailed usage description check `python docker_build_test.py --help`.
+- Sanity tests for the docker image are present in test/docker_sanity_test.py.
+- By default image will be built and tested, but if you only want to build the image, pass `-b` flag and if you only want to test the given image pass `-t` flag.
+- An html test report will be generated after the tests are executed containing the results.
+
+Bulding image and running tests using github actions
+----------------------------------------------------
+This is the recommended way to build, test and get a CVE report for the docker image.
+Just choose the image type and provide kafka url to `Docker build test` workflow. It will generate a test report and CVE report that can be shared to the community.
+
+Creating a release
+------------------
+- `docker_release.py` script builds a multi architecture image and pushes it to provided docker registry.
+- Ensure you are logged in to the docker registry before triggering the script.
+- kafka binary tarball url along with image name (in the format `<registry>/<namespace>/<image_name>:<image_tag>`) and type is needed to build the image. For detailed usage description check `python docker_release.py --help`.
+
+Promoting a release
+-------------------
+`docker_promote.py` provides an interactive way to pull an RC Docker image and promote it to required dockerhub repo.
+
+Using the image in a docker container
+-------------------------------------
+- The image uses the kafka downloaded from provided kafka url
+- The image can be run in a container in default mode by running
+`docker run <image-name:tag> -p 9092:9092`
```

Comment:
I am getting this error with -p flag

 ```
 docker run kafka-master:latest -p 9092:9092
/__cacert_entrypoint.sh: line 30: exec: -p: invalid option
exec: usage: exec [-cl] [-a name] [command [argument ...]] [redirection ...]
```

Reply:
Ah thanks for catching this. It's indeed incorrect. I have fixed it in the last commit. Please try `docker run -p 9092:9092 kafka-master:latest`